components/landing/DemoQuestion.tsx

```
@@ -0,0 +1,127 @@
+  const handleAnswerSelect = (label: 'A' | 'B' | 'C' | 'D') => {
+    if (showResult) return // Prevent answer changes after submission
+    
+    setSelectedAnswer(label)
+    // Auto-submit after selection
+    setShowResult(true)
+  }
+
+  const isCorrect = selectedAnswer === question.correctAnswer
+
+  return (
+    <div ref={ref as any} className="w-full max-w-3xl mx-auto px-4">
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)